tests/gocase/util/server.go

```diff
@@ -181,11 +238,19 @@ func StartServer(t testing.TB, configs map[string]string) *KvrocksServer {
 
 	c := redis.NewClient(&redis.Options{Addr: addr.String()})
 	defer func() { require.NoError(t, c.Close()) }()
+
+	proc, err := process.NewProcess(int32(cmd.Process.Pid))
+	require.NoError(t, err)
+
+	procStatus := []string{}
 	require.Eventually(t, func() bool {
 		err := c.Ping(context.Background()).Err()
-		return err == nil || err.Error() == "NOAUTH Authentication required."
+		procStatus, _ = proc.Status()
+		return err == nil || err.Error() == "NOAUTH Authentication required." || reflect.DeepEqual(procStatus, []string{process.Zombie})
```

Comment:
If kvrocks aborted due to an config error or whatever, we can exit the ping loop quickly rather than waiting 1 minute.